README.md

```diff
@@ -129,19 +130,11 @@ For example:
 
 - `weights`
 A list of weights, each weights definition contains the following fields:
-    - `id` a unique id which will be used to refer to the weights. <!-- maybe with special values like 'default'? -->
-    - `name` the name of the weights for display, it should be a human-friendly name in title case
-    - `description` description about the weights, it is recommended to describe the how the weights is trained, and what's the dataset used for training.
+    - `weight_format` format of weight entry
     - `authors` a list of authors. This field is optional, only required if the authors are different from the model.
-    - `covers` a list of cover images (see `model`:`covers`). This is used for showing how inputs and outputs look like with this weights file.
     - `source` link to the weights file. Preferably an url to the weights file.
     - `sha256` SHA256 checksum of the model weight file specified by `source` (see `models` section above for how to generate SHA256 checksum)
     - `timestamp` timestamp according to [ISO 8601](#https://en.wikipedia.org/wiki/ISO_8601)
-    - `test_inputs` list of URIs to test inputs as described in `inputs` for a single test case. Supported file formats/extensions: `.npy`
-    - `test_outputs` analog to `test_inputs`.
-    - `documentation` relative path to file with additional documentation in markdown.
-    - `tags` a list of tags.
-    - `attachments` text keys and URI values to additional, relevant files.
```

Comment:
We can keep the attachments here for attaching files with this specific weights, the attachments will be used for automatic model packaging.

Reply:
I think the `attachments` should be global. I don't see a reason to have them separately for the different weights.